tools/bazelify_tests/build_defs.bzl

```diff
@@ -204,3 +246,102 @@ def grpc_run_simple_command_test(name, args = [], data = [], size = "medium", ti
 
     env = {}
     _dockerized_sh_test(name = name, srcs = srcs, args = args, data = data, size = size, timeout = timeout, tags = tags, exec_compatible_with = exec_compatible_with, flaky = flaky, docker_image_version = docker_image_version, env = env, docker_run_as_root = False)
+
+def grpc_build_artifact_task(name, timeout = None, artifact_deps = [], tags = [], exec_compatible_with = [], flaky = None, docker_image_version = None, build_script = None):
+    """Execute a build artifact task and a corresponding 'build test'
```

Comment:
nit: this needs a short description of how it relies upon the genrule (which always succeeds).

Reply:
Done.